spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -357,6 +357,7 @@ def assert_flavors
         expect(flavor.enabled).to               eq true
         expect(flavor.cpu_cores).to             eq nil
         expect(flavor.description).to           eq descr
+        expect(flavor.type).to                  eq "ManageIQ::Providers::Openstack::CloudManager::Flavor"
```

Comment:
These ensure that the auto_model_class is doing the right thing, if it fails it will be `::Flavor`.